doc/sphinx-guides/source/developers/making-releases.rst

```diff
@@ -58,7 +58,7 @@ At or near release time:
 - The "tag version" and "title" should be the number of the milestone with a "v" in front (i.e. v4.16).
 - Copy in the content from the .md file
 - For the description, follow post-4.16 examples at https://github.com/IQSS/dataverse/releases
-
+- Delete the .md file in the source code. This is to ensure that only one copy of the notes lives on in the future, as a single source of truth.
```

Comment:
@scolapasta this is the line I mentioned in standup that we might want to think about. That is, when does it make sense to remove the combined release note file from git?

I think of "merge develop into master" as the final step before a release with regard to commits. And since the release note file is temporary, we should probably remove it before this final step.